views/pages/index.ejs

```diff
@@ -22,22 +22,69 @@
 
     <!-- Tailwind css cdn  -->
     <script src="https://cdn.tailwindcss.com"></script>
+
+    <style>
```

Comment:
Same thing please donot write styles in the `views` files. We are maintaining a proper directory structure. Please write the scripts in `public/css` directory and link them in `views`